src/opm/io/eclipse/rst/state.cpp

```diff
@@ -350,12 +350,12 @@ void RstState::add_actions(const Parser& parser,
 void RstState::add_wlist(const std::vector<std::string>& zwls,
                          const std::vector<int>& iwls)
 {
-    for (std::size_t well_index = 0; well_index < this->header.num_wells; well_index++) {
+    for (int well_index = 0; well_index < this->header.num_wells; well_index++) {
```

Comment:
I've recently switched to writing these kinds of loops as
```C++
for (auto i = 0*n; i < n; ++i)
```
since this always produces an induction variable of the correct type, initialised with the value zero.